### PR TITLE
docs(contributing): move to docs directory [skip ci]

### PR DESCRIPTION
**Pull request by boopathi**

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Debugging
+
+In your project, if you find that there is a bug that appears ONLY when you use BabelMinify, it's most likely that there is a bug in BabelMinify and you should definitely report it. Here are some guidelines that might help you drill down the issue. If it doesn't help you, you can of course create a minimal repro project with the bug and report it.
```
Comment (vigneshshanmugam) [inline docs/debugging.md]: Can we extend this and add like
`when you use BabelMinify standalone or babel-minify as a preset in babelrc`

File: docs/debugging.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Compile time Errors
+
+If you get a syntax error at compile time, then it could be a few things:
+
+1. The parser itself doesn't handle the syntax being used (a [babylon](https://github.com/babel/babylon) bug).
```
Comment (j-f1) [inline docs/debugging.md]: Babylon is now inside the monorepo.

File: docs/debugging.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+2. The code is actually invalid syntax.
+3. You didn't turn on the relevant Babel plugin for that syntax (if experimental).
```
Comment (j-f1) [inline docs/debugging.md]: Should this be in order by probability? (2-3-1)

File: docs/tests.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+# Tests
+
+## Lint
+
+This project uses [prettier](https://github.com/prettier/prettier) for formatting code and [eslint](https://github.com/eslint/eslint) for other linting.
```
Comment (j-f1) [inline docs/tests.md]: ESLint :)

File: docs/tests.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+We use [jest](https://github.com/facebook/jest) for testing and follow the following directory structure -
+
+```
+packages/
+  - babel-plugin-minify-mangle-names/
+    - __tests__/
+      - fixtures/
+        - fixture-name/
+          - actual.js                   # source
+          - expected.js                 # minify transforms applied (except whitespace)
```
Comment (j-f1) [inline docs/tests.md]: and comment removal

**Pull request merged**